ui_framework/doc_site/src/views/accessibility/screen_reader.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default () => (
+  <div>
+    <p>
+      This is the first paragraph. It is visible to all.
+    </p>
+    <p className='kuiScreenReaderOnly'>
```

Comment:
I could, but would that be very useful? As a CSS class, it can be applied on any kind of DOM element, but as a React element, it would provide an unnecessary wrapper node. Maybe there is a trick that I don't know though :)